STACK-UP-3/champs-backend
# #171648173 Feature allows users to sign in using Google and Facebook

#### Description
A feature allows users to sign-in using their Google account(Gmail) and Facebook account
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test
Sign in using existing Gmail or Facebook account
Give instructions on how reviewers can verify your work

#### Any background context you want to add (Optional)

Screenshots or GIFs

#### STORY ID

[Finishes #171648173]
